map/map_loader/README.md

```diff
@@ -154,10 +159,11 @@ The node projects lan/lon coordinates into MGRS coordinates.
 
 ### Feature
 
-lanelet2_map_visualization visualizes autoware_auto_mapping_msgs/HADMapBin messages into visualization_msgs/MarkerArray. There are 3 types of map can be loaded in autoware. Please make sure you selected the correct lanelet2_map_projector_type when you launch this package.
+lanelet2_map_visualization visualizes autoware_auto_mapping_msgs/HADMapBin messages into visualization_msgs/MarkerArray. There are four types of map can be loaded in autoware. Please make sure you selected the correct lanelet2_map_projector_type when you launch this package.
 
 - MGRS
 - UTM
+- TransverseMercator
 - local
```

Comment:
This isn't your fault, but description about projection should go under explanation for lanelet2_map_loader [at L140](https://github.com/autowarefoundation/autoware.universe/blob/cd09c426cb1cbb2c80d7ab56e7408930b71c5438/map/map_loader/README.md?plain=1#L140), and not under visualization because it doesn't do any projection in visualization node.

```suggestion
lanelet2_map_visualization visualizes autoware_auto_mapping_msgs/HADMapBin messages into visualization_msgs/MarkerArray. 
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/3986/commits/4b8ddbbb0bd9f0870cd822993b0d3fc44c866fbe